templates/types/streaming/nextjs/app/components/ui/chat/index.ts

```diff
@@ -30,7 +31,19 @@ export type EventData = {
   isCollapsed: boolean;
 };
 
-export type AnnotationData = ImageData | SourceData | EventData;
+export type ToolData = {
+  toolCall: {
+    id: string;
+    name: string;
+    input: any;
+  };
+  toolOutput: {
+    output: any;
```

Comment:
JSONValue for any?

Reply:
yes, it's different, but here's it's already streamed by vercel, so it should be the `JSONValue` type from vercel/ai?